pages/ksphere/konvoy/latest/release-notes/index.md

```diff
@@ -223,13 +265,13 @@ kind: ClusterConfiguration
 apiVersion: konvoy.mesosphere.io/v1alpha1
 spec:
   kubernetes:
-    version: 1.16.4
+    version: 1.16.8
   containerNetworking:
     calico:
       version: v3.10.1
   addons:
     configRepository: https://github.com/mesosphere/kubernetes-base-addons
-    configVersion: stable-1.16.4-2
+    configVersion: stable-1.16.8-2
```

Comment:
this doesn't exist, should we leave as `stable-1.16.4-2`?

Reply:
Reverted this, this shouldn't have been changed.